pkg/controlplane/controller/apiserverleasegc/gc_controller.go

```diff
@@ -73,9 +73,9 @@ func NewAPIServerLeaseGC(clientset kubernetes.Interface, gcCheckPeriod time.Dura
 // Run starts one worker.
 func (c *Controller) Run(stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
-	defer klog.Infof("Shutting down apiserver lease garbage collector")
+	defer klog.InfoS("Shutting down API Server lease garbage collector")
```

Comment:
minor: We have `API Server`, `API server`, `apiserver` combined in code base. Most of the customer facing messages were using `API server`. e.g. https://github.com/kubernetes/kubernetes/blob/1c1527f5198b4e8391ce37082dbe166879750295/staging/src/k8s.io/client-go/tools/clientcmd/overrides.go#L180

Reply:
Fixed to "API server".